_includes/manuals/2.0/1.2_release_notes.md

```diff
@@ -4,6 +4,18 @@ This section will be updated prior to the next release.
 
 ### Headline features
 
+#### New sidekiq-based implementation for Dynflow
+
+#### Consolidating on PostgreSQL as only supported database
+In the past, Foreman was trying to be database-agnostic and support multiple databases. That has proven to be very challenging and led to multiple issues due to differences between databases. Starting with Foreman 2.0, we are consolidating on PostgreSQL as the only supported database. In the near future Katello will also switch to using Pulp 3, meaning that Katello users will also enjoy a single database instead of needing to manage multiple ones. RPM based installations will also upgrade PostgreSQL to use version 12 from SCL instead of the default version 9 shipped in EL7 repositories.
```

Comment:
3.15 will already ship Pulp 3 for file and container content types so that future is very near (shipping together with 2.0).

EL7 shipped with 9.2

Reply:
thanks, updated to clarify.